presto-main/src/main/java/com/facebook/presto/twitter/TwitterModuleLoader.java

```diff
@@ -1,64 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.facebook.presto.twitter;
```

Comment:
The main code we'd like to track is the logic to parse the event to build the thrift object. If that could survive in git that would be great. If that's hard to do we can remove it and add the relevant parts back.